docs/specification.md

```diff
@@ -251,7 +250,6 @@ The list of errors may be inaccurate and incomplete, it will be updated during t
 | `NOT_PENDING_ADMIN`             | Authorized sender is not the current pending administrator                                                  |
 | `NOT_TOKEN_OWNER`               | Trying to configure operators for a different wallet which sender does not own                              |
 | `FAIL_PROPOSAL_CHECK`           | Throws when trying to propose a proposal that does not pass `proposalCheck`                                 |
-| `FROZEN_TOKEN_NOT_TRANSFERABLE` | Transfer entrypoint is called for frozen token by a non-admin sender                                        |
```

Comment:
Following the previous comment, this error still needs to exist, the difference is that the `by a non-admin sender` needs to be removed, as it now applies to that role as well.